### DISCUSSION TITLE
Welcome to application-tracking-system Discussions!

### DISCUSSION THREAD

**Discussion by sachindoddaguni** [sachindoddaguni/application-tracking-system#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sachindoddaguni** [comment]
Adding a skill builder section would be useful to the user.

**Comment by sachindoddaguni** [comment]
I mean we can provide the references to courses when searched for a specific skill
Could be something for future scope

**Comment by amoghmahesh14** [comment]
Yeah that's a good idea!
We can also try to integrate this application with the levels.fyi website to provide salary information

**Comment by sachindoddaguni** [comment]
Thats a good thought. Would be useful to have salary information within the portal. Might provide extra motivation to the user to apply to specific jobs.

**Comment by amoghmahesh14** [comment]
Does that website provide publicly available free APIs?
